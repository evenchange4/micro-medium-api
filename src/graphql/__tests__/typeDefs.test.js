const typeDefs = require('../typeDefs');

it('should return typeDefs', () => {
  expect(typeDefs).toMatchSnapshot();
});
