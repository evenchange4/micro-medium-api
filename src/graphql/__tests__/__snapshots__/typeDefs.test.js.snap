// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return typeDefs 1`] = `
"
  scalar Timestamp

  type Content {
    subtitle: String
  }

  type Post {
    id: ID!
    title: String!
    uniqueSlug: String!
    url: String!
    content: Content!
    firstPublishedAt: Timestamp!
    latestPublishedAt: Timestamp
    createdAt: Timestamp
    updatedAt: Timestamp
  }

  type User {
    userId: ID!
    name: String!
    username: String!
    createdAt: Timestamp,
    lastPostCreatedAt: Timestamp,
    imageId: String
    backgroundImageId: String
    bio: String
    type: String
  }

  type Query {
    posts(username: String!, limit: Int = 10): [Post!]!
    user(username: String!): User!
  }
"
`;
